src/test/kotlin/io/github/thewisenerd/linters/sidekt/ResourceOnboardedOnApsecTest.kt

```diff
@@ -0,0 +1,46 @@
+package io.github.thewisenerd.linters.sidekt
+import io.github.thewisenerd.linters.sidekt.rules.ResourceOnboardedOnApsec
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.Finding
+import io.gitlab.arturbosch.detekt.api.SourceLocation
+import io.gitlab.arturbosch.detekt.test.compileAndLintWithContext
+import org.junit.Test
+
+class ResourceOnboardedOnApsecTest {
+
+    companion object {
+        private val blockingJerseyMethod = ResourceOnboardedOnApsec::class.java.simpleName
```

Comment:
I think the variable name 'blockingJerseyMethod' was mistakenly put. Please change it to the appropriate name.